uniflow/op/extract/split/pattern_splitter_op.py

```diff
@@ -8,15 +11,24 @@
 class PatternSplitter(Op):
     """Pattern Splitter Op Class"""
 
-    def __init__(self, name: str, splitter: str = "\n\n") -> None:
+    default_separators = "\n\n|\n"
+    default_min_chunk_size = 1
+
+    def __init__(self, splitterConfig: dict, name: str = "paragraph_split_op") -> None:
```

Comment:
nit: will be nicer to have name as first argument and Config as second to consistent with other Ops.